tests/ExtendedPdoTest.php

```diff
@@ -651,4 +651,18 @@ public function testWithProfileLogging()
         $this->pdo->query("SELECT 5 FROM pdotest");
         $this->assertEquals(4, count($logger->getLog()));
     }
+
+    public function testNoExportOfLoginCredentials()
+    {
+        $pdo = new ExtendedPdo('sqlite::memory:', 'username', 'password');
+        $debugInfo = $pdo->__debugInfo();
```

Comment:
Also maybe it would be better to see what properties are printed by `var_dump` prior to `__debugInfo` method being added and then return values from these properties (with credentials stipped). Currently you might be removing non-sensitive info, that was shown in `var_dump` result prior to this PR.